test/edu/ucsb/nceas/metacat/dataone/CNodeServiceTest.java

```diff
@@ -223,8 +224,12 @@ public void testCreate() {
 			guid.setValue("testCreate." + System.currentTimeMillis());
 			InputStream object = new ByteArrayInputStream("test".getBytes("UTF-8"));
 			SystemMetadata sysmeta = createSystemMetadata(guid, session.getSubject(), object);
+			Date originalModificationDate = sysmeta.getDateSysMetadataModified();
+			Thread.sleep(1000);
```

Comment:
is there any other alternative to doing this? It's buggy, and it makes the tests take longer to run

Reply:
defer for now - will address in issue #1702